dotnet-desktop-guide/net/winforms/get-started/create-app-visual-studio.md

```diff
@@ -0,0 +1,163 @@
+- [Visual Studio Desktop workload](/visualstudio/install/modify-visual-studio?view=vs-2019&preserve-view=true)
+
+## Create a WinForms app
+
+The first step to creating a new app is opening Visual Studio and generating the app from a template.
+
+01. Open Visual Studio.
+01. Select **Create a new project**.
+
+    :::image type="content" source="media/create-app-visual-studio/vs-create-new-project.png" alt-text="Create a new Windows Forms project in Visual Studio 2019 for .NET":::
+
+01. In the **Search for templates** box, type **winforms**, and then press <kbd>Enter</kbd>.
+01. In the **code language** dropdown, choose **C#** or **Visual Basic**.
+01. The templates list, select **Windows Forms App (.NET)** and then click **Next**.
+
+    > [!IMPORTANT]
+    > Don't select the **Windows Forms App (.NET _Framework_)** template.
+
+    :::image type="content" source="media/create-app-visual-studio/vs-template-search.png" alt-text="Search for the Windows Forms template in Visual Studio 2019 for .NET":::
+
+01. In the **Configure your new project** window, set the **Project name** to **Names** and click **Create**.
+
+    You can also save your project to a different folder by adjusting the **Location** setting.
+
+    :::image type="content" source="media/create-app-visual-studio/vs-config-new-project.png" alt-text="Configure new Windows Forms project in Visual Studio 2019 for .NET":::
+
+Once the app is generated, Visual Studio should open the designer pane for the default form, _Form1_. If the form designer isn't visible, double-click on the form in the **Solution explorer** pane to open the designer window.
+
+### Important parts of Visual Studio
+
+Support for WinForms in Visual Studio has four important components that you'll interact with as you create an app:
+
+01. Solution explorer
+
+    All if your project files, code, forms, resources, will appear in this pane.
+
+02. Properties
+
+    This pane shows property settings you can configure based on the item selected. For example, if you select an item from the **Solution explorer** you'll see property settings related to the file. If you select an object in the **Designer** you'll see settings for the control or form.
+
+03. Form Designer
+
+    This is the designer for the form. It's interactive and you can drag-and-drop objects from the **Toolbox**. By selecting and moving items in the designer, you can visually compose the user interface (UI) for your app.
+
+04. Toolbox
+
+    The toolbox contains all of the controls you can add to a form. To add a control to the current form, double-click a control or drag-and-drop the control.
+
+    :::image type="content" source="media/create-app-visual-studio/vs-main-window.png" alt-text="The important components of Visual Studio you should know when creating a Windows Forms project for .NET":::
```

Comment:
I think you should move the image to come before the 1-4 descriptions (line 61). Also, at the end of the descriptions, you might offer a link to more info about VS, e.g. https://docs.microsoft.com/en-us/visualstudio/get-started/visual-studio-ide?view=vs-2019.